annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -141,6 +144,7 @@ protected void validateTagWithSpecificResource(Body body) throws ParamValidation
      */
     protected void validateTranscriptionWithFullTextResource(Body body)
 	    throws ParamValidationException, PropertyValidationException, RequestBodyValidationException {
+    if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.